src/main/java/uk/gov/hmcts/probate/service/ccd/CcdClientApi.java

```diff
@@ -87,10 +89,43 @@ public CaseDetails updateCaseAsCaseworker(CcdCaseType caseType, String caseId, C
                 caseId,
                 eventId.getName()
         );
-        CaseDataContent caseDataContent = createCaseDataContent(caseData, eventId, startEventResponse);
+        CaseDataContent caseDataContent = createCaseDataContent(caseData, eventId, startEventResponse,
+                PROBATE_APPLICATION, PROBATE_APPLICATION);
         log.info("Submit event to CCD for Caseworker, caseType: {}, caseId: {}",
                 caseType.getName(), caseId);
-        CaseDetails caseDetails = coreCaseDataApi.submitEventForCaseWorker(
+        return coreCaseDataApi.submitEventForCaseWorker(
+                securityDTO.getAuthorisation(),
+                securityDTO.getServiceAuthorisation(),
+                securityDTO.getUserId(),
+                PROBATE.name(),
+                caseType.getName(),
+                caseId,
+                false,
+                caseDataContent
+        );
+    }
+
+    @Override
+    public CaseDetails updateCaseAsCitizen(CcdCaseType caseType, String caseId, CaseData caseData, EventId eventId,
+                                              SecurityDTO securityDTO, String description, String summary) {
+        log.info("Update case as citizen for caseType: {}, caseId: {}, eventId: {}",
+                caseType.getName(), caseId, eventId.getName());
+        log.info("Retrieve event token from CCD for citizen, caseType: {}, caseId: {}, eventId: {}",
```

Comment:
can we add lev keyword too in this message to make it easy to associate to lev calls

Reply:
Added an additional log message to LifeEventService. 
This method already logs the eventId:

2021-May-12 13:21:59.819 INFO  [task-3] u.g.h.p.s.c.CcdClientApi - Update case as citizen for caseType: GrantOfRepresentation, caseId: 1620825628357285, eventId: deathRecordVerified

Reply:
Removed redundant log info